docs/rst/tutorials/cloud/edge_edge_repeater_cloud/edge_edge_repeater_cloud.rst

```diff
@@ -0,0 +1,346 @@
+
+Running DDS Router Edge 1
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Then, to run the DDS Router Edge configure the environment as in the previous step.
+
+.. note::
+
+    If you are deploying Vulcanexus from the Docker image, note that you will need to have a configuration file (``config.yaml``) to configure the DDS Router Edge accessible from your Docker container.
+
+    You can do this by mounting a shared volume when launching the container, by copying the file from the local host to the container in case it is already running, or by editing a file from the Docker container itself.
+
+Setup the Vulcanexus environment, either in a Docker container or on the local host, running the following command:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+
+Let's create the DDS Router configuration file. It will look like the one shown below.
+
+.. literalinclude:: /resources/tutorials/cloud/edge_edge_repeater_cloud/dds_router_edge_1.yaml
+    :language: yaml
+
+Next, it is briefly explained the most relevant aspects of this configuration file.
+
+The ``participants`` are the interfaces of the DDS Router to communicate with other networks. In this case, we have two participants:
+
+    *   ``local``: this is a simple participant that communicates with all ROS 2 nodes it finds in domain 1.
+        For more information about this participant please refer to the `Simple Participant section <https://eprosima-dds-router.readthedocs.io/en/latest/rst/user_manual/participants/simple.html#user-manual-participants-simple>`_ of the DDS Router documentation.
+
+    *   ``repeater-client``: it is a participant designed to communicate with a Repeater Participant.
+        It uses the `Fast DDS Initial Peers discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html#initial-peers>`_ to establish a point-to-point communication between two DDS entities, two *DDS Routers* in this case.
+
+    For the DDS Router Edge, a connection address shall be defined which must be the same as the one exposed by the Cloud Server.
+    There are some relevant configurations within this connection address:
+
+    * ``connection-addresses``: defines the IP (``ip``) and port (``port``) of the network addresses to which it connects, and the transport protocol (``transport``) to be used in the communication, TCP in this case.
+
+.. note::
+
+    In this case, the DDS Router will forward all topics found in the network.
+    However, it is important to mention that the ROS 2 topics relayed by the DDS Router can be filtered by configuring the ``allowlist`` and ``blocklist``.
+    If this is the case please refer to the `DDS Router documentation <https://eprosima-dds-router.readthedocs.io/en/latest/>`_ for information on how to do this.
+
+The following figure summarizes the deployment on the edge.
+
+.. todo::
+
+    add new image
+
+.. figure:: /rst/figures/tutorials/cloud/edge_1_repeater_deployment.png
```

Comment:
Again, make sure the same domain is being used.

Reply:
I do not understand the suggestion. In the image domain 0 is used

Reply:
See above.